week08_pomdp/practice_pytorch.ipynb

```diff
@@ -591,25 +661,110 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "for i in trange(15000):\n",
-    "\n",
-    "    memory = list(pool.prev_memory_states)\n",
-    "    rollout_obs, rollout_actions, rollout_rewards, rollout_mask = pool.interact(\n",
-    "        10)\n",
-    "    train_on_rollout(rollout_obs, rollout_actions,\n",
-    "                     rollout_rewards, rollout_mask, memory)\n",
-    "\n",
-    "    if i % 100 == 0:\n",
-    "        rewards_history.append(np.mean(evaluate(agent, env, n_games=1)))\n",
-    "        clear_output(True)\n",
-    "        plt.plot(rewards_history, label='rewards')\n",
-    "        plt.plot(moving_average(np.array(rewards_history),\n",
-    "                                span=10), label='rewards ewma@10')\n",
-    "        plt.legend()\n",
-    "        plt.show()\n",
-    "        if rewards_history[-1] >= 10000:\n",
-    "            print(\"Your agent has just passed the minimum homework threshold\")\n",
-    "            break"
+    "rollout_len = 10\n",
+    "# Change to higher number of steps after you ensure you get progress\n",
+    "STEPS = int(5e+4) #int(15e+3)\n",
```

Comment:
`5e+4` and `15e+3` is a lot harder to read than `50_000` and `15_000`. Please fix.